integreat_cms/cms/templates/ajax_poi_form/poi_box.html

```diff
@@ -11,17 +11,24 @@
     {{ title }}
 {% endblock collapsible_box_title %}
 {% block collapsible_box_content %}
-    {% render_field form.has_not_location class+='inline-block' %}
-    <label class="secondary !inline"
-           for="{{ form.has_not_location.id_for_label }}">
-        {{ form.has_not_location.label }}
-    </label>
+    {% if current_menu_item == "events_form" %}
+        {% render_field form.has_not_location class+='inline-block' %}
+        <label class="secondary !inline"
+               for="{{ form.has_not_location.id_for_label }}">
+            {{ form.has_not_location.label }}
+        </label>
+    {% endif %}
+    {% if current_menu_item == "contacts" %}
+        <div class="help-text mt-4 font-bold">
+            {{ help_text }}
+        </div>
+    {% endif %}
```

Comment:
Since this was designed as a generic component, we should not include checkes here from where this is used imo. In this case, could you just move this code to the places from where the poi box is included?

Reply:
I know what I mean 😓 Though those elements have to stay outside of the box if they are put in the event form or contat form before or after {% include "poi_box.html" %} , or otherwise we need to reform the box itself 🤔 Do you see any handy fix idea?

Reply:
Agree 😁 #3068